packages/cli/src/index.ts

```diff
@@ -15,11 +15,11 @@
  */
 
 import {Cli} from './lib/Cli';
-import {Log} from '@bubblewrap/core';
+import {Log, ConsoleLog} from '@bubblewrap/core';
 
 module.exports = async (): Promise<void> => {
   const cli = new Cli();
-  const log = new Log('cli');
+  const log: Log = new ConsoleLog('cli');
```

Comment:
So while we do want to change the types of parameters to `Log`, we don't really care about explicitly changing the type of local variables to `Log`.